src/server/utils.ts

```diff
@@ -284,7 +284,7 @@ export const validateBreadcrumbs = (breadcrumbs: Breadcrumb[]) => {
 
 // Validator utils
 export const isNavUrl = (url: string) =>
-    /^(\/|(https?:\/\/localhost)|(https:\/\/([a-z0-9-]+[.])*nav[.]no($|\/)))/i.test(url);
+    /^(\/|(https?:\/\/localhost)|(https:\/\/([a-z0-9-]+[.])*((nav[.]no)|(nais[.]io))))/i.test(url);
```

Comment:
```suggestion
    /^(\/|(https?:\/\/localhost)|(https:\/\/([a-z0-9-]+[.])*((nav[.]no)|(nais[.]io))($|\/)))/i.test(url);
```

Må ha med den end-string eller forward-slash sjekken på slutten for å hindre at url'er type https://nav.noevilscammer.com valideres